### ISSUE TITLE
Adding RTSP stream to Homebridge

### ISSUE THREAD

**Issue by WEZANGO** [TheCrypt0/yi-hack-v4#66]
Did anybody managed to add RTSP stream from Yi Hack V4 to homebridge? I can watch the stream through VLC but in HomeKit it's only black screen without any video. 

I am using regular sapmle config provided at homebridge ffmpeg plugin wiki

`{
  "platform": "Camera-ffmpeg",
  "cameras": [
    {
      "name": "Camera Name",
      "videoConfig": {
        "source": "-re -i rtsp://myfancy_rtsp_stream",
        "stillImageSource": "-i rtsp://myfancy_rtsp_stream",
        "maxStreams": 2,
        "maxWidth": 1280,
        "maxHeight": 720,
        "maxFPS": 30
      }
    }
  ]
}`

**Comment by SeydX** [comment]
yes, i have modified the homebridge-cameras-ffmpeg plugin and using also the mqtt service from this hack

**Comment by xvolte** [comment]
@SeydX can you share what you did please?

**Comment by WEZANGO** [comment]
> yes, i have modified the homebridge-cameras-ffmpeg plugin and using also the mqtt service from this hack

Are you using Raspberry Pi for that or something else?

**Comment by SeydX** [comment]
@WEZANGO 

yea its a raspberry

@xvolte 

will publish a own plugin for this within next few days

**Comment by SeydX** [comment]
v1 

https://github.com/SeydX/homebridge-camera-mqtt

**Comment by WEZANGO** [comment]
So it is an Mqtt plugin. But how did you add camera itself? Actually I already managed to add my camera, but it connects in about 10 secınds and it has really low frame rate, freezes once in a few seconds. I would say it’s because raspberry is not really powerfull, but while watching camera, CPU is loaded only for 20%. Do you have the same issues?

**Comment by SeydX** [comment]
@WEZANGO 

Its a ffmpeg (its also possible to use other codecs) plugin with mqtt support designed for the mqtt service which this hack offers

Im using this with a Intel NUC, with the example-config you can find on the main site of the plugin

Do you tried with ffmpeg?

**Comment by SeydX** [comment]
changed from homebridge-camera-mqtt to homebridge-yi-camera

here some screenshots

https://github.com/SeydX/homebridge-yi-camera/tree/master/images

**Comment by WEZANGO** [comment]
> 
> 
> changed from homebridge-camera-mqtt to homebridge-yi-camera
> 
> here some screenshots
> 
> https://github.com/SeydX/homebridge-yi-camera/tree/master/images

That is just great. Thank a lot!

**Comment by SeydX** [comment]
Next version will add also a web interface for the camera to access the stream via http from overall

**Comment by chriseckman** [comment]
@SeydX What changes did you need to make? I'm having similar issue with Shinobi.
